### PR TITLE
News 9.3 - Config Files

### PR DESCRIPTION
**Pull request by alexsuter**
More ideas?
What can/should I link to?

### CODE REVIEW

File: src/app/pages/news/9.3/02-config-files.md [hunk 1]
```diff
@@ -1,19 +1,8 @@
...
-	</a>
-	<a href="${docBaseUrl}/public-api/ch/ivyteam/ivy/rest/client/oauth2/OAuth2BearerFilter.html" target="_blank" rel="noopener noreferrer">
-		<i class="si si-movie"></i> Public API: OAuth2
-	</a>
-</div>
+The entire configuration in Axon Ivy projects has been revised and consolidated.
+This will make the daily life for all developers much easier.
 
+- __Streamlining__ : The configuration of rest clients, web service clients, databases or variables now looks exactly the same in your Axon Ivy project as on the Axon Ivy Engine.
+- __Known file extensions__ : All configuration files now have standard file extensions which gives you code hihghlighting in all other tools and also when sending a pull request to a friend.
```
Comment (ivy-rew) [inline src/app/pages/news/9.3/02-config-files.md:7]: typo: hihghlighting

File: src/app/pages/news/9.3/02-config-files.md [hunk 2]
```diff
@@ -1,19 +1,8 @@
...
-		<i class="si si-check"></i> OpenAPI converters
-	</a>
-	<a href="${docBaseUrl}/public-api/ch/ivyteam/ivy/rest/client/oauth2/OAuth2BearerFilter.html" target="_blank" rel="noopener noreferrer">
-		<i class="si si-movie"></i> Public API: OAuth2
-	</a>
-</div>
+The entire configuration in Axon Ivy projects has been revised and consolidated.
+This will make the daily life for all developers much easier.
 
+- __Streamlining__ : The configuration of rest clients, web service clients, databases or variables now looks exactly the same in your Axon Ivy project as on the Axon Ivy Engine.
```
Comment (ivy-rew) [inline src/app/pages/news/9.3/02-config-files.md:6]: another idea: ready to customize by copying into app.yaml ... make these configs sexy not only for developers, but operations too

File: src/app/pages/news/9.3/02-config-files.md [hunk 3]
```diff
@@ -1,19 +1,8 @@
 ## Config files {#project93}
 
-... SCM ready, easy DevOps, new tree, ...
-
-<div class="short-links">
```
Comment (ivy-rew) [inline src/app/pages/news/9.3/02-config-files.md:5]: linking is indeed a good question: maybe we should start with a file-reference as we do for engine configs e.g.
https://dev.axonivy.com/doc/nightly/engine-guide/configuration/files/app-yaml.html
would fit nicely into 
https://dev.axonivy.com/doc/nightly/designer-guide/configuration/index.html
... but just an idea: we can do it now or later

File: src/app/pages/news/9.3/02-config-files.md [hunk 4]
```diff
@@ -1,19 +1,8 @@
 ## Config files {#project93}
```
Comment (ivy-rew) [inline src/app/pages/news/9.3/02-config-files.md:2]: give us a screenshot of the new sexy config files: e.g. showing partially the new project tree ...
plus an standard editor + the same resource opened in a split view using the .yaml editor?

**Review by ivy-rew — APPROVED** [review]

**Comment by alexsuter** [comment]
grafik

**Comment by ivy-rew** [comment]
sehr nice :1st_place_medal:

**Pull request merged**